protected/humhub/modules/user/models/Password.php

```diff
@@ -83,8 +83,7 @@ public function rules()
             [['newPassword', 'newPasswordConfirm', 'currentPassword'], 'required', 'on' => 'changePassword'],
```

Comment:
`['newPassword', 'newPasswordConfirm', 'currentPassword'],` could be only `['currentPassword'],` . In first rule `newPassword', 'newPasswordConfirm', ` checks by `required validator`